app/controllers/api/invitations_controller.rb

```diff
@@ -5,7 +5,11 @@ def create
                                                      group: @group,
                                                      inviter: current_user,
                                                      message: invitation_form_params[:message])
-    respond_with_collection
+    if @invitations.empty? and email_addresses.any?
```

Comment:
As it stands it shouldn't get to the server unless there are email addresses, since we disable the form. I'd be happy with a generic error message like

'We weren't able to send any invitations to the emails provided. They have been added to the group already.'